cirq/google/json_test_data/spec.py

```diff
@@ -0,0 +1,37 @@
+import pathlib
+
+import cirq
+from cirq.google.json_resolver_cache import _class_resolver_dictionary
+
+from cirq.testing.json import ModuleJsonTestSpec
+
+TestSpec = ModuleJsonTestSpec(name="cirq.google",
+                              packages=[cirq.google],
+                              test_data_path=pathlib.Path(__file__).parent,
+                              not_yet_serializable=[
+                                  'CalibrationResult', 'CalibrationLayer',
+                                  'FSIM_GATESET', 'SYC_GATESET', 'Sycamore',
+                                  'Sycamore23', 'SerializableDevice',
+                                  'SerializableGateSet', 'SQRT_ISWAP_GATESET',
+                                  'XmonDevice', 'XMON'
+                              ],
+                              should_not_be_serialized=[
+                                  'Engine',
```

Comment:
We could have an internal `@_NotJsonSerializable` annotation to localize this information better.

I'm a bit wary of using annotations to opt-in, because then there's no central place to see the list of what can be parsed, but for opt-out it's fine.

Reply:
Good idea - could be a follow-up PR just to not bloat this one further?

Reply:
I forgot to open this issue, and forgot about the idea and then re-invented it :) I'll work on it after this. https://github.com/quantumlib/Cirq/issues/3734